ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/metrics/DelayMetricSpec.scala

```diff
@@ -11,6 +11,8 @@ import org.scalatest.wordspec.AnyWordSpec
 import java.time.{Clock, Duration, Instant, ZoneId}
 import scala.util.Random
 
+import scala.language.existentials
```

Comment:
Are you using existential types?

Reply:
Yes, specifically when calling `.violatedObjectives` method.